openapi.yaml

```diff
@@ -0,0 +1,473 @@
+      security:
+      - UserSecurity: []
+    delete:
+      tags:
+      - "post"
+      summary: "Remove action for the post"
+      operationId: "RemoveActionPost"
+      parameters:
+      - name: "id"
+        in: "path"
+        description: "Post id"
+        required: true
+        type: "string"
+      - name: "action"
+        in: "path"
+        description: "Post id"
+        required: true
+        type: "string"
+        enum:
+          - "save"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/Post"
+        default:
+          description: "error operation"
+          schema:
+            $ref: "#/definitions/ApiErrorResponse"
+      security:
+      - UserSecurity: []
+
+securityDefinitions:
+  UserSecurity:
+    type: basic
+
+definitions:
+  User:
+    type: "object"
+    required:
+    - "email"
+    properties:
+      id:
+        type: "string"
+        description: "Id for the user"
+      email:
+        type: "string"
+        description: "Email for the user"
+      token:
+        type: "string"
+        description: "Token for the user when login by email"
+      vendor:
+        type: "string"
+        description: "Vendro for the auth"
+      sources:
+        type: "array"
+        items:
+          type: "string"
+          description: "Reference to source model"
+      profile:
+        type: "object"
+        properties:
+          id:
+            type: "string"
+            description: "Facebook id"
+          username:
+            type: "string"
+            description: "Facebook username"
+          displayName:
+            type: "string"
+            description: "Facebook display name"
+          name:
+            type: "object"
+            properties:
+              familyName:
+                type: "string"
+                description: "Facebook first name"
+              giveName:
+                type: "string"
+                description: "Facebook last name"
+              middleName:
+                type: "string"
+                description: "Facebook middle name"
+          gender:
+            type: "string"
+            description: "Facebook gender"
+          profileUrl:
+            type: "string"
+            description: "Facebook profile url"
+          emails:
+            type: "array"
+            description: "Facebook email list"
+            items:
+              type: "string"
+          photos:
+            type: "array"
+            description: "Facebook photo list"
+            items:
+              type: "string"
+  Source:
+    type: "object"
+    required:
+    - "title"
+    - "url"
+    properties:
+      id:
+        type: "string"
+        description: "Id for the source"
+      title:
+        type: "string"
+        description: "Title for the source"
+      url:
+        type: "string"
+        description: "Url for the source"
```

Comment:
remove the url. the url means the rss url of the source for scraping data. the client does not need it.